src/main/webapp/app/exercises/programming/manage/programming-exercise.component.html

```diff
@@ -244,18 +244,15 @@ <h4 class="mb-1 me-auto">
                             </div>
                             <div class="btn-group-vertical me-1">
                                 <button
-                                    *ngIf="programmingExercise.isAtLeastInstructor"
+                                    *ngIf="programmingExercise.isAtLeastEditor"
                                     [jhiFeatureToggle]="FeatureToggle.ProgrammingExercises"
-                                    jhiDeleteButton
-                                    [actionType]="ActionType.Reset"
-                                    [entityTitle]="programmingExercise.title!"
-                                    deleteQuestion="artemisApp.programmingExercise.reset.question"
-                                    (delete)="resetProgrammingExercise(programmingExercise.id!)"
-                                    [dialogError]="dialogError$"
-                                    deleteConfirmationText="artemisApp.exercise.delete.typeNameToConfirm"
+                                    jhiProgrammingExerciseResetButton
+                                    [programmingExercise]="programmingExercise"
                                     class="mb-1"
+                                    id="reset-exercise"
```

Comment:
Is it never the case that `programming-exercise-detail.component` and this `programming-exercise.component` are used simultaneously somehow? Otherwise there could be a problem here with the HTML element ID clash (in the other component, there's also a button with `id="reset-exercise"`. If you are sure that's fine you can just resolve this.

Reply:
True, I removed it. Thanks!